lib/iris/tests/unit/analysis/test_MIN.py

```diff
@@ -4,21 +4,20 @@
 # See LICENSE in the root of the repository for full licensing details.
 """Unit tests for the :data:`iris.analysis.MIN` aggregator."""
 
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests  # isort:skip
-
 import numpy as np
 import numpy.ma as ma
+import pytest
 
 from iris._lazy_data import as_lazy_data, is_lazy_data
 from iris.analysis import MIN
 from iris.coords import DimCoord
 from iris.cube import Cube
+from iris.tests import _shared_utils
 
 
-class Test_basics(tests.IrisTest):
-    def setUp(self):
+class Test_basics:
+    @pytest.fixture(autouse=True)
+    def _setup(self):
```

Comment:
This setup block seems to be duplicated across multiple files (thanks pycharm). 

That makes it a pretty good candidate for a fixture, strange though it may sound.